### ISSUE TITLE
The Nature of the Firm

### ISSUE THREAD

**Issue by tieubao** [tieubao/til#371]
https://onlinelibrary.wiley.com/doi/full/10.1111/j.1468-0335.1937.tb00002.x

-----

Economic theory has suffered in the past from a failure to state clearly its assumptions. Economists in building up a theory have often omitted to examine the foundations on which it was erected. This examination is, however, essential not only to prevent the misunderstanding and needless controversy which arise from a lack of knowledge of the assumptions on which a theory is based, but also because of the extreme importance for economics of good judgment in choosing between rival sets of assumptions. For instance, it is suggested that the use of the word “firm” in economics may be different from the use of the term by the “plain man.”1 Since there is apparently a trend in economic theory towards starting analysis with the individual firm and not with the industry,2 it is all the more necessary not only that a clear definition of the word “firm” should be given but that its difference from a firm in the “real world,” if it exists, should be made clear. Mrs. Robinson has said that “the two questions to be asked of a set of assumptions in economics are : Are they tractable? and : Do they correspond with the real world?”3 Though, as Mrs. Robinson points out, “more often one set will be manageable and the other realistic,” yet there may well be branches of theory where assumptions may be both manageable and realistic. It is hoped to show in the following paper that a definition of a firm may be obtained which is not only realistic in that it corresponds to what is meant by a firm in the real world, but is tractable by two of the most powerful instruments of economic analysis developed by Marshall, the idea of the margin and that of substitution, together giving the idea of substitution at the margin.1 Our definition must, of course, “relate to formal relations which are capable of being conceived exactly.”2

I
It is convenient if, in searching for a definition of a firm, we first consider the economic system as it is normally treated by the economist. Let us consider the description of the economic system given by Sir Arthur Salter.3“The normal economic system works itself. For its current operation it is under no central control, it needs no central survey. Over the whole range of human activity and human need, supply is adjusted to demand, and production to consumption, by a process that is automatic, elastic and responsive.” An economist thinks of the economic system as being co‐ordinated by the price mechanism and society becomes not an organisation but an organism.4 The economic system “works itself.” This does not mean that there is no planning by individuals. These exercise foresight and choose between alternatives. This is necessarily so if there is to be order in the system. But this theory assumes that the direction of resources is dependent directly on the price mechanism. Indeed, it is often considered to be an objection to economic planning that it merely tries to do what is already done by the price mechanism.5 Sir Arthur Salter's description, however, gives a very incomplete picture of our economic system. Within a firm, the description does not fit at all. For instance, in economic theory we find that the allocation of factors of production between different uses is determined by the price mechanism. The price of factor A becomes higher in X than in Y. As a result, A moves from Y to X until the difference between the prices in X and Y, except in so far as it compensates for other differential advantages, disappears. Yet in the real world, we find that there are many areas where this does not apply. If a workman moves from department Y to department X, he does not go because of a change in relative prices, but because he is ordered to do so. Those who object to economic planning on the grounds that the problem is solved by price movements can be answered by pointing out that there is planning within our economic system which is quite different from the individual planning mentioned above and which is akin to what is normally called economic planning. The example given above is typical of a large sphere in our modern economic system. Of course, this fact has not been ignored by economists. Marshall introduces organisation as a fourth factor of production; J. B. Clark gives the co‐ordinating function to the entrepreneur; Professor Knight introduces managers who co‐ordinate. As D. H. Robertson points out, we find “islands of conscious power in this ocean of unconscious co‐operation like lumps of butter coagulating in a pail of buttermilk.”1 But in view of the fact that it is usually argued that co‐ordination will be done by the price mechanism, why is such organisation necessary? Why are there these “islands of conscious power”? Outside the firm, price movements direct production, which is co‐ordinated through a series of exchange transactions on the market. Within a firm, these market transactions are eliminated and in place of the complicated market structure with exchange transactions is substituted the entrepreneur‐co‐ordinator, who directs production.2 It is clear that these are alternative methods of co‐ordinating production. Yet, having regard to the fact that if production is regulated by price movements, production could be carried on without any organisation at all, well might we ask, why is there any organisation?

Of course, the degree to which the price mechanism is superseded varies greatly. In a department store, the allocation of the different sections to the various locations in the building may be done by the controlling authority or it may be the result of competitive price bidding for space. In the Lancashire cotton industry, a weaver can rent power and shop‐room and can obtain looms and yarn on credit.3 This co‐ordination of the various factors of production is, however, normally carried out without the intervention of the price mechanism. As is evident, the amount of “vertical” integration, involving as it does the supersession of the price mechanism, varies greatly from industry to industry and from firm to firm.

It can, I think, be assumed that the distinguishing mark of the firm is the supersession of the price mechanism. It is, of course, as Professor Robbins points out, “related to an outside network of relative prices and costs,”1 but it is important to discover the exact nature of this relationship. This distinction between the allocation of resources in a firm and the allocation in the economic system has been very vividly described by Mr. Maurice Dobb when discussing Adam Smith's conception of the capitalist: “It began to be seen that there was something more important than the relations inside each factory or unit captained by an undertaker; there were the relations of the undertaker with the rest of the economic world outside his immediate sphere…. the undertaker busies himself with the division of labour inside each firm and he plans and organises consciously,” but “he is related to the much larger economic specialisation, of which he himself is merely one specialised unit. Here, he plays his part as a single cell in a larger organism, mainly unconscious of the wider rôle he fills.”2

In view of the fact that while economists treat the price mechanism as a co‐ordinating instrument, they also admit the co‐ordinating function of the “entrepreneur,” it is surely important to enquire why co‐ordination is the work of the price mechanism in one case and of the entrepreneur in another. The purpose of this paper is to bridge what appears to be a gap in economic theory between the assumption (made for some purposes) that resources are allocated by means of the price mechanism and the assumption (made for other purposes) that this allocation is dependent on the entrepreneur‐co‐ordinator. We have to explain the basis on which, in practice, this choice between alternatives is effected.3

II
Our task is to attempt to discover why a firm emerges at all in a specialised exchange economy. The price mechanism (considered purely from the side of the direction of resources) might be superseded if the relationship which replaced it was desired for its own sake. This would be the case, for example, if some people preferred to work under the direction of some other person. Such individuals would accept less in order to work under someone, and firms would arise naturally from this. But it would appear that this cannot be a very important reason, for it would rather seem that the opposite tendency is operating if one judges from the stress normally laid on the advantage of “being one's own master.”1 Of course, if the desire was not to be controlled but to control, to exercise power over others, then people might be willing to give up something in order to direct others; that is, they would be willing to pay others more than they could get under the price mechanism in order to be able to direct them. But this implies that those who direct pay in order to be able to do this and are not paid to direct, which is clearly not true in the majority of cases.2 Firms might also exist if purchasers preferred commodities which are produced by firms to those not so produced; but even in spheres where one would expect such preferences (if they exist) to be of negligible importance, firms are to be found in the real world.3 Therefore there must be other elements involved.

The main reason why it is profitable to establish a firm would seem to be that there is a cost of using the price mechanism. The most obvious cost of “organising” production through the price mechanism is that of discovering what the relevant prices are.4 This cost may be reduced but it will not be eliminated by the emergence of specialists who will sell this information. The costs of negotiating and concluding a separate contract for each exchange transaction which takes place on a market must also be taken into account.1 Again, in certain markets, e.g., produce exchanges, a technique is devised for minimising these contract costs; but they are not eliminated. It is true that contracts are not eliminated when there is a firm but they are greatly reduced. A factor of production (or the owner thereof) does not have to make a series of contracts with the factors with whom he is co‐operating within the firm, as would be necessary, of course, if this co‐operation were as a direct result of the working of the price mechanism. For this series of contracts is substituted one. At this stage, it is important to note the character of the contract into which a factor enters that is employed within a firm. The contract is one whereby the factor, for a certain remuneration (which may be fixed or fluctuating), agrees to obey the directions of an entrepreneur within certain limits.2 The essence of the contract is that it should only state the limits to the powers of the entrepreneur. Within these limits, he can therefore direct the other factors of production.

There are, however, other disadvantages—or costs—of using the price mechanism. It may be desired to make a long‐term contract for the supply of some article or service. This may be due to the fact that if one contract is made for a longer period, instead of several shorter ones, then certain costs of making each contract will be avoided. Or, owing to the risk attitude of the people concerned, they may prefer to make a long rather than a short‐term contract. Now, owing to the difficulty of forecasting, the longer the period of the contract is for the supply of the commodity or service, the less possible, and indeed, the less desirable it is for the person purchasing to specify what the other contracting party is expected to do. It may well be a matter of indifference to the person supplying the service or commodity which of several courses of action is taken, but not to the purchaser of that service or commodity. But the purchaser will not know which of these several courses he will want the supplier to take. Therefore, the service which is being provided is expressed in general terms, the exact details being left until a later date. All that is stated in the contract is the limits to what the persons supplying the commodity or service is expected to do. The details of what the supplier is expected to do is not stated in the contract but is decided later by the purchaser. When the direction of resources (within the limits of the contract) becomes dependent on the buyer in this way, that relationship which I term a “firm” may be obtained.1 A firm is likely therefore to emerge in those cases where a very short term contract would be unsatisfactory. It is obviously of more importance in the case of services—labour—than it is in the case of the buying of commodities. In the case of commodities, the main items can be stated in advance and the details which will be decided later will be of minor significance.

We may sum up this section of the argument by saying that the operation of a market costs something and by forming an organisation and allowing some authority (an “entrepreneur”) to direct the resources, certain marketing costs are saved. The entrepreneur has to carry out his function at less cost, taking into account the fact that he may get factors of production at a lower price than the market transactions which he supersedes, because it is always possible to revert to the open market if he fails to do this.

The question of uncertainty is one which is often considered to be very relevant to the study of the equilibrium of the firm. It seems improbable that a firm would emerge without the existence of uncertainty. But those, for instance, Professor Knight, who make the mode of payment the distinguishing mark of the firm—fixed incomes being guaranteed to some of those engaged in production by a person who takes the residual, and fluctuating, income—would appear to be introducing a point which is irrelevant to the problem we are considering. One entrepreneur may sell his services to another for a certain sum of money, while the payment to his employees may be mainly or wholly a share in profits.2 The significant question would appear to be why the allocation of resources is not done directly by the price mechanism.

Another factor that should be noted is that exchange transactions on a market and the same transactions organised within a firm are often treated differently by Governments or other bodies with regulatory powers. If we consider the operation of a sales tax, it is clear that it is a tax on market transactions and not on the same transactions organised within the firm. Now since these are alternative methods of “organisation”—by the price mechanism or by the entrepreneur—such a regulation would bring into existence firms which otherwise would have no raison d'être. It would furnish a reason for the emergence of a firm in a specialised exchange economy. Of course, to the extent that firms already exist, such a measure as a sales tax would merely tend to make them larger than they would otherwise be. Similarly, quota schemes, and methods of price control which imply that there is rationing, and which do not apply to firms producing such products for themselves, by allowing advantages to those who organise within the firm and not through the market, necessarily encourage the growth of firms. But it is difficult to believe that it is measures such as have been mentioned in this paragraph which have brought firms into existence. Such measures would, however, tend to have this result if they did not exist for other reasons.

These, then, are the reasons why organisations such as firms exist in a specialised exchange economy in which it is generally assumed that the distribution of resources is “organised” by the price mechanism. A firm, therefore, consists of the system of relationships which comes into existence when the direction of resources is dependent on an entrepreneur.

The approach which has just been sketched would appear to offer an advantage in that it is possible to give a scientific meaning to what is meant by saying that a firm gets larger or smaller. A firm becomes larger as additional transactions (which could be exchange transactions co‐ordinated through the price mechanism) are organised by the entrepreneur and becomes smaller as he abandons the organisation of such transactions. The question which arises is whether it is possible to study the forces which determine the size of the firm. Why does the entrepreneur not organise one less transaction or one more? It is interesting to note that Professor Knight considers that:

“the relation between efficiency and size is one of the most serious problems of theory, being, in contrast with the relation for a plant, largely a matter of personality and historical accident rather than of intelligible general principles. But the question is peculiarly vital because the possibility of monopoly gain offers a powerful incentive to continuous and unlimited expansion of the firm, which force must be offset by some equally powerful one making for decreased efficiency (in the production of money income) with growth in size, if even boundary competition is to exist.”1
Professor Knight would appear to consider that it is impossible to treat scientifically the determinants of the size of the firm. On the basis of the concept of the firm developed above, this task will now be attempted.

It was suggested that the introduction of the firm was due primarily to the existence of marketing costs. A pertinent question to ask would appear to be (quite apart from the monopoly considerations raised by Professor Knight), why, if by organising one can eliminate certain costs and in fact reduce the cost of production, are there any market transactions at all?2 Why is not all production carried on by one big firm? There would appear to be certain possible explanations.

First, as a firm gets larger, there may be decreasing returns to the entrepreneur function, that is, the costs of organising additional transactions within the firm may rise.3 Naturally, a point must be reached where the costs of organising an extra transaction within the firm are equal to the costs involved in carrying out the transaction in the open market, or, to the costs of organising by another entrepreneur. Secondly, it may be that as the transactions which are organised increase, the entrepreneur fails to place the factors of production in the uses where their value is greatest, that is, fails to make the best use of the factors of production. Again, a point must be reached where the loss through the waste of resources is equal to the marketing costs of the exchange transaction in the open market or to the loss if the transaction was organised by another entrepreneur. Finally, the supply price of one or more of the factors of production may rise, because the “other advantages” of a small firm are greater than those of a large firm.1 Of course, the actual point where the expansion of the firm ceases might be determined by a combination of the factors mentioned above. The first two reasons given most probably correspond to the economists' phrase of “diminishing returns to management.”2

The point has been made in the previous paragraph that a firm will tend to expand until the costs of organising an extra transaction within the firm become equal to the costs of carrying out the same transaction by means of an exchange on the open market or the costs of organising in another firm. But if the firm stops its expansion at a point below the costs of marketing in the open market and at a point equal to the costs of organising in another firm, in most cases (excluding the case of “combination”3), this will imply that there is a market transaction between these two producers, each of whom could organise it at less than the actual marketing costs. How is the paradox to be resolved? If we consider an example the reason for this will become clear. Suppose A is buying a product from B and that both A and B could organise this marketing transaction at less than its present cost. B, we can assume, is not organising one process or stage of production, but several. If A therefore wishes to avoid a market transaction, he will have to take over all the processes of production controlled by B. Unless A takes over all the processes of production, a market transaction will still remain, although it is a different product that is bought. But we have previously assumed that as each producer expands he becomes less efficient; the additional costs of organising extra transactions increase. It is probable that A's cost of organising the transactions previously organised by B will be greater than B's cost of doing the same thing. A therefore will take over the whole of B's organisation only if his cost of organising B's work is not greater than B's cost by an amount equal to the costs of carrying out an exchange transaction on the open market. But once it becomes economical to have a market transaction, it also pays to divide production in such a way that the cost of organising an extra transaction in each firm is the same.

Up to now it has been assumed that the exchange transactions which take place through the price mechanism are homogeneous. In fact, nothing could be more diverse than the actual transactions which take place in our modern world. This would seem to imply that the costs of carrying out exchange transactions through the price mechanism will vary considerably as will also the costs of organising these transactions within the firm. It seems therefore possible that quite apart from the question of diminishing returns the costs of organising certain transactions within the firm may be greater than the costs of carrying out the exchange transactions in the open market. This would necessarily imply that there were exchange transactions carried out through the price mechanism, but would it mean that there would have to be more than one firm? Clearly not, for all those areas in the economic system where the direction of resources was not dependent directly on the price mechanism could be organised within one firm. The factors which were discussed earlier would seem to be the important ones, though it is difficult to say whether “diminishing returns to management” or the rising supply price of factors is likely to be the more important.

Other things being equal, therefore, a firm will tend to be larger:

(a)
 the less the costs of organising and the slower these costs rise with an increase in the transactions organised.

(b)
 the less likely the entrepreneur is to make mistakes and the smaller the increase in mistakes with an increase in the transactions organised.

(c)
 the greater the lowering (or the less the rise) in the supply price of factors of production to firms of larger size.

Apart from variations in the supply price of factors of production to firms of different sizes, it would appear that the costs of organising and the losses through mistakes will increase with an increase in the spatial distribution of the transactions organised, in the dissimilarity of the transactions, and in the probability of changes in the relevant prices.3 As more transactions are organised by an entrepreneur, it would appear that the transactions would tend to be either different in kind or in different places. This furnishes an additional reason why efficiency will tend to decrease as the firm gets larger. Inventions which tend to bring factors of production nearer together, by lessening spatial distribution, tend to increase the size of the firm.2 Changes like the telephone and the telegraph which tend to reduce the cost of organising spatially will tend to increase the size of the firm. All changes which improve managerial technique will tend to increase the size of the firm.1

It should be noted that the definition of a firm which was given above can be used to give more precise meanings to the terms “combination” and “integration.”5 There is a combination when transactions which were previously organised by two or more entrepreneurs become organised by one. This becomes integration when it involves the organisation of transactions which were previously carried out between the entrepreneurs on a market. A firm can expand in either or both of these two ways. The whole of the “structure of competitive industry” becomes tractable by the ordinary technique of economic analysis.

III
The problem which has been investigated in the previous section has not been entirely neglected by economists and it is now necessary to consider why the reasons given above for the emergence of a firm in a specialised exchange economy are to be preferred to the other explanations which have been offered.

It is sometimes said that the reason for the existence of a firm is to be found in the division of labour. This is the view of Professor Usher, a view which has been adopted and expanded by Mr. Maurice Dobb. The firm becomes “the result of an increasing complexity of the division of labour…. The growth of this economic differentiation creates the need for some integrating force without which differentiation would collapse into chaos; and it is as the integrating force in a differentiated economy that industrial forms are chiefly significant.”1 The answer to this argument is an obvious one. The “integrating force in a differentiated economy” already exists in the form of the price mechanism. It is perhaps the main achievement of economic science that it has shown that there is no reason to suppose that specialisation must lead to chaos.2 The reason given by Mr. Maurice Dobb is therefore inadmissible. What has to be explained is why one integrating force (the entrepreneur) should be substituted for another integrating force (the price mechanism).

The most interesting reasons (and probably the most widely accepted) which have been given to explain this fact are those to be found in Professor Knight's Risk, Uncertainty and Profit. His views will be examined in some detail.

Professor Knight starts with a system in which there is no uncertainty:

“acting as individuals under absolute freedom but without collusion men are supposed to have organised economic life with the primary and secondary division of labour, the use of capital, etc., developed to the point familiar in present‐day America. The principal fact which calls for the exercise of the imagination is the internal organisation of the productive groups or establishments. With uncertainty entirely absent, every individual being in possession of perfect knowledge of the situation, there would be no occasion for anything of the nature of responsible management or control of productive activity. Even marketing transactions in any realistic sense would not be found. The flow of raw materials and productive services to the consumer would be entirely automatic.”1
Professor Knight says that we can imagine this adjustment as being “the result of a long process of experimentation worked out by trial‐and‐error methods alone,” while it is not necessary “to imagine every worker doing exactly the right thing at the right time in a sort of ‘pre‐established harmony’ with the work of others. There might be managers, superintendents, etc., for the purpose of co‐ordinating the activities of individuals,” though these managers would be performing a purely routine function, “without responsibility of any sort.”2

Professor Knight then continues:

“With the introduction of uncertainty—the fact of ignorance and the necessity of acting upon opinion rather than knowledge—into this Eden‐like situation, its character is entirely changed…. With uncertainty present doing things, the actual execution of activity, becomes in a real sense a secondary part of life; the primary problem or function is deciding what to do and how to do it.”3
This fact of uncertainty brings about the two most important characteristics of social organisation.

“In the first place, goods are produced for a market, on the basis of entirely impersonal prediction of wants, not for the satisfaction of the wants of the producers themselves. The producer takes the responsibility of forecasting the consumers' wants. In the second place, the work of forecasting and at the same time a large part of the technological direction and control of production are still further concentrated upon a very narrow class of the producers, and we meet with a new economic functionary, the entrepreneur…. When uncertainty is present and the task of deciding what to do and how to do it takes the ascendancy over that of execution the internal organisation of the productive groups is no longer a matter of indifference or a mechanical detail. Centralisation of this deciding and controlling function is imperative, a process of ‘cephalisation’ is inevitable.”1
The most fundamental change is:

“the system under which the confident and venturesome assume the risk or insure the doubtful and timid by guaranteeing to the latter a specified income in return for an assignment of the actual results…. With human nature as we know it it would be impracticable or very unusual for one man to guarantee to another a definite result of the latter's actions without being given power to direct his work. And on the other hand the second party would not place himself under the direction of the first without such a guarantee…. The result of this manifold specialisation of function is the enterprise and wage system of industry. Its existence in the world is the direct result of the fact of uncertainty.”2
These quotations give the essence of Professor Knight's theory. The fact of uncertainty means that people have to forecast future wants. Therefore, you get a special class springing up who direct the activities of others to whom they give guaranteed wages. It acts because good judgment is generally associated with confidence in one's judgment.3

Professor Knight would appear to leave himself open to criticism on several grounds. First of all, as he himself points out, the fact that certain people have better judgment or better knowledge does not mean that they can only get an income from it by themselves actively taking part in production. They can sell advice or knowledge. Every business buys the services of a host of advisers. We can imagine a system where all advice or knowledge was bought as required. Again, it is possible to get a reward from better knowledge or judgment not by actively taking part in production but by making contracts with people who are producing. A merchant buying for future delivery represents an example of this. But this merely illustrates the point that it is quite possible to give a guaranteed reward providing that certain acts are performed without directing the performance of those acts. Professor Knight says that “with human nature as we know it it would be impracticable or very unusual for one man to guarantee to another a definite result of the latter's actions without being given power to direct his work.” This is surely incorrect. A large proportion of jobs are done to contract, that is, the contractor is guaranteed a certain sum providing he performs certain acts. But this does not involve any direction. It does mean, however, that the system of relative prices has been changed and that there will be a new arrangement of the factors of production.1 The fact that Professor Knight mentions that the “second party would not place himself under the direction of the first without such a guarantee” is irrelevant to the problem we are considering. Finally, it seems important to notice that even in the case of an economic system where there is no uncertainty Professor Knight considers that there would be co‐ordinators, though they would perform only a routine function. He immediately adds that they would be “without responsibility of any sort,” which raises the question by whom are they paid and why? It seems that nowhere does Professor Knight give a reason why the price mechanism should be superseded.

IV
It would seem important to examine one further point and that is to consider the relevance of this discussion to the general question of the “cost‐curve of the firm.”

It has sometimes been assumed that a firm is limited in size under perfect competition if its cost curve slopes upward,2 while under imperfect competition, it is limited in size because it will not pay to produce more than the output at which marginal cost is equal to marginal revenue.1 But it is clear that a firm may produce more than one product and, therefore, there appears to be no prima facie reason why this upward slope of the cost curve in the case of perfect competition or the fact that marginal cost will not always be below marginal revenue in the case of imperfect competition should limit the size of the firm.2 Mrs. Robinson3 makes the simplifying assumption that only one product is being produced. But it is clearly important to investigate how the number of products produced by a firm is determined, while no theory which assumes that only one product is in fact produced can have very great practical significance.

It might be replied that under perfect competition, since everything that is produced can be sold at the prevailing price, then there is no need for any other product to be produced. But this argument ignores the fact that there may be a point where it is less costly to organise the exchange transactions of a new product than to organise further exchange transactions of the old product. This point can be illustrated in the following way. Imagine, following von Thunen, that there is a town, the consuming centre, and that industries are located around this central point in rings. These conditions are illustrated in the following diagram in which A, B and C represent different industries. Imagine an entrepreneur who starts controlling exchange transactions from x. Now as he extends his activities in the same product (B), the cost of organising increases until at some point it becomes equal to that of a dissimilar product which is nearer. As the firm expands, it will therefore from this point include more than one product (A and C). This treatment of the problem is obviously incomplete,1 but it is necessary to show that merely proving that the cost curve turns upwards does not give a limitation to the size of the firm. So far we have only considered the case of perfect competition; the case of imperfect competition would appear to be obvious.

To determine the size of the firm, we have to consider the marketing costs (that is, the costs of using the price mechanism), and the costs of organising of different entrepreneurs and then we can determine how many products will be produced by each firm and how much of each it will produce. It would, therefore, appear that Mr. Shove2 in his article on “Imperfect Competition” was asking questions which Mrs. Robinson's cost curve apparatus cannot answer. The factors mentioned above would seem to be the relevant ones.

V
Only one task now remains; and that is, to see whether the concept of a firm which has been developed fits in with that existing in the real world. We can best approach the question of what constitutes a firm in practice by considering the legal relationship normally called that of “master and servant” or “employer and employee.”3 The essentials of this relationship have been given as follows:

1
 “the servant must be under the duty of rendering personal services to the master or to others on behalf of the master, otherwise the contract is a contract for sale of goods or the like.

2
 The master must have the right to control the servant's work, either personally or by another servant or agent. It is this right of control or interference, of being entitled to tell the servant when to work (within the hours of service) and when not to work, and what work to do and how to do it (within the terms of such service) which is the dominant characteristic in this relation and marks off the servant from an independent contractor, or from one employed merely to give to his employer the fruits of his labour. In the latter case, the contractor or performer is not under the employer's control in doing the work or effecting the service; he has to shape and manage his work so as to give the result he has contracted to effect.”1

We thus see that it is the fact of direction which is the essence of the legal concept of “employer and employee,” just as it was in the economic concept which was developed above. It is interesting to note that Professor Batt says further:

“That which distinguishes an agent from a servant is not the absence or presence of a fixed wage or the payment only of commission on business done, but rather the freedom with which an agent may carry out his employment”2
We can therefore conclude that the definition we have given is one which approximates closely to the firm as it is considered in the real world.

Our definition is, therefore, realistic. Is it manageable? This ought to be clear. When we are considering how large a firm will be the principle of marginalism works smoothly. The question always is, will it pay to bring an extra exchange transaction under the organising authority? At the margin, the costs of organising within the firm will be equal either to the costs of organising in another firm or to the costs involved in leaving the transaction to be “organised” by the price mechanism. Business men will be constantly experimenting, controlling more or less, and in this way, equilibrium will be maintained. This gives the position of equilibrium for static analysis. But it is clear that the dynamic factors are also of considerable importance, and an investigation of the effect changes have on the cost of organising within the firm and on marketing costs generally will enable one to explain why firms get larger and smaller. We thus have a theory of moving equilibrium. The above analysis would also appear to have clarified the relationship between initiative or enterprise and management. Initiative means forecasting and operates through the price mechanism by the making of new contracts. Management proper merely reacts to price changes, rearranging the factors of production under its control. That the business man normally combines both functions is an obvious result of the marketing costs which were discussed above. Finally, this analysis enables us to state more exactly what is meant by the “marginal product” of the entrepreneur. But an elaboration of this point would take us far from our comparatively simple task of definition and clarification.

Footnotes
1  Joan Robinson, Economics is a Serious Subject, p. 12.
2  See N. Kaldor, “The Equilibrium of the Firm,”Economic Journal, March, 1934.
3  Op. cit., p. 6.
1  J. M. Keynes, Essays in Biography, pp. 223–4.
2  L. Robbins, Nature and Significance of Economic Science, p. 63.
3  This description is quoted with approval by D. H. Robertson, Control of Industry, p. 85, and by Professor Arnold Plant, “Trends in Business Administration,” Economica, February, 1932. It appears in Allied Shipping Control, pp. 16–17.
4  See F. A. Hayek, “The Trend of Economic Thinking,” Economica, May, 1933.
5  See F. A. Hayek, op. cit.
1  Op. cit., p. 85.
2  In the rest of this paper I shall use the term entrepreneur to refer to the person or persons who, in a competitive system, take the place of the price mechanism in the direction of resources.
3  Survey of Textile Industries, p. 26.
1  Op. cit., p. 71.
2  Capitalist Enterprise and Social Progress, p. 20. Cf., also, Henderson, Supply and Demand, pp. 3–5.
3  It is easy to see when the State takes over the direction of an industry that, in planning it, it is doing something which was previously done by the price mechanism. What is usually not realised is that any business man in organising the relations between his departments is also doing something which could be organised through the price mechanism. There is therefore point in Mr. Durbin's answer to those who emphasise the problems involved in economic planning that the same problems have to be solved by business men in the competitive system. (See “Economic Calculus in a Planned Economy,”Economic Journal, December, 1936.) The important difference between these two cases is that economic planning is imposed on industry while firms arise voluntarily because they represent a more efficient method of organising production. In a competitive system, there is an “optimum” amount of planning!
1  Cf. Harry Dawes, “Labour Mobility in the Steel Industry,”Economic Journal, March, 1934, who instances “the trek to retail shopkeeping and insurance work by the better paid of skilled men due to the desire (often the main aim in life of a worker) to be independent” (p. 86).
2  None the less, this is not altogether fanciful. Some small shopkeepers are said to earn less than their assistants.
3  G. F. Shove, “The Imperfection of the Market: a Further Note,”Economic Journal, March, 1933, p. 116, note 1, points out that such preferences may exist, although the example he gives is almost the reverse of the instance given in the text.
4  According to N. Kaldor, “A Classificatory Note of the Determinateness of Equilibrium,”Review of Economic Studies, February, 1934, it is one of the assumptions of static theory that “All the relevant prices are known to all individuals.” But this is clearly not true of the real world.
1  This influence was noted by Professor Usher when discussing the development of capitalism. He says: “The successive buying and selling of partly finished products were sheer waste of energy.” (Introduction to the Industrial History of England, p. 13). But he does not develop the idea nor consider why it is that buying and selling operations still exist.
2  It would be possible for no limits to the powers of the entrepreneur to be fixed. This would be voluntary slavery. According to Professor Batt, The Law of Master and Servant, p. 18, such a contract would be void and unenforceable.
1  Of course, it is not possible to draw a hard and fast line which determines whether there is a firm or not. There may be more or less direction. It is similar to the legal question of whether there is the relationship of master and servant or principal and agent. See the discussion of this problem below.
2  The views of Professor Knight are examined below in more detail.
1  Risk, Uncertainty and Profit, Preface to the Re‐issue, London School of Economics Series of Reprints, No. 16, 1933.
2  There are certain marketing costs which could only be eliminated by the abolition of “consumers' choice” and these are the costs of retailing. It is conceivable that these costs might be so high that people would be willing to accept rations because the extra product obtained was worth the loss of their choice.
3  This argument assumes that exchange transactions on a market can be considered as homogeneous; which is clearly untrue in fact. This complication is taken into account below.
1  For a discussion of the variation of the supply price of factors of production to firms of varying size, see E. A. G. Robinson, The Structure of Competitive Industry. It is sometimes said that the supply price of organising ability increases as the size of the firm increases because men prefer to be the heads of small independent businesses rather than the heads of departments in a large business. See Jones, The Trust Problem, p. 531, and Macgregor, Industrial Combination, p. 63. This is a common argument of those who advocate Rational‐sation. It is said that larger units would be more efficient, but owing to the individualistic spirit of the smaller entrepreneurs, they prefer to remain independent, apparently in spite of the higher income which their increased efficiency under Rationalisation makes possible.
2  This discussion is, of course, brief and incomplete. For a more thorough discussion of this particular problem, see N. Kaldor, “The Equilibrium of the Firm,”Economic Journal, March, 1934, and E. A. G. Robinson, “The Problem of Management and the Size of the Firm,”Economic Journal, June, 1934.
3  A definition of this term is given below.
This aspect of the problem is emphasised by N. Kaldor, op. cit. Its importance in this connection had been previously noted by F. A. G. Robinson, The Structure of Competitive Industry, pp. 83–106. This assumes that an increase in the probability of price movements increases the costs of organising within a firm more than it increases the cost of carrying out an exchange transaction on the market—which is probable.
2  This would appear to be the importance of the treatment of the technical unit by E. A. G. Robinson, op. cit., pp. 27–33. The larger the technical unit, the greater the concentration of factors and therefore the firm is likely to be larger.
3  It should be noted that most inventions will change both the costs of organising and the costs of using the price mechanism. In such cases, whether the invention tends to make firms larger or smaller will depend on the relative effect on these two sets of costs. For instance, if the telephone reduces the costs of using the price mechanism more than it reduces the costs of organising, then it will have the effect of reducing the size of the firm.
4  An illustration of these dynamic forces is furnished by Maurice Dobb, Russian Economic Development, p. 68. “With the passing of bonded labour the factory, as an establishment where work was organised under the whip of the overseer, lost its raison d'être until this was restored to it with the introduction of power machinery after 1846.” It seems important to realise that the passage from the domestic system to the factory system is not a mere historical accident, but is conditioned by economic forces. This is shown by the fact that it is possible to move from the factory system to the domestic system, as in the Russian example, as well as vice versa. It is the essence of serfdom that the price mechanism is not allowed to operate. Therefore, there has to be direction from some organiser. When, however, serfdom passed, the price mechanism was allowed to operate. It was not until machinery drew workers into one locality that it paid to supersede the price mechanism and the firm again emerged.
5  This is often called “vertical integration,” combination being termed “lateral integration.”
1  Op. cit., p. 10. Professor Usher's views are to be found in his Introduction to the Industrial History of England, pp. 1–18.
2  Cf. J. B. Clark, Distribution of Wealth, p. 19, who speaks of the theory of exchange as being the “theory of the organisation of industrial society.”
1  Risk, Uncertainty and Profit, p. 267.
2  Op. cit., pp. 267–8.
3  Op. cit., p. 268.
1  Op. cit., pp. 268–95.
2  Op. cit., pp. 269–70.
3  Op. cit., p. 270.
1  This shows that it is possible to have a private enterprise system without the existence of firms. Though, in practice, the two functions of enterprise, which actually influences the system of relative prices by forecasting wants and acting in accordance with such forecasts, and management, which accepts the system of relative prices as being given, are normally carried out by the same persons, yet it seems important to keep them separate in theory. This point is further discussed below.
2  See Kaldor, op. cit., and Robinson, The Problem of Management and the Size of the Firm.
1  Mr. Robinson calls this the Imperfect Competition solution for the survival of the small firm.
2  Mr. Robinson's conclusion, op. cit., p. 249, note 1, would appear to be definitely wrong. He is followed by Horace J. White, Jr., “Monopolistic and Perfect Competition,”American Economic Review, December, 1936, p. 645, note 27. Mr. White states “It is obvious that the size of the firm is limited in conditions of monopolistic competition.”
3  Economics of Imperfect Competition.
1  As has been shown above, location is only one of the factors influencing the cost of organising.
2  G. F. Shove, “The Imperfection of the Market,”Economic Journal, March, 1933, p. 115. In connection with an increase in demand in the suburbs and the effect on the price charged by suppliers, Mr. Shove asks “…. why do not the old firms open branches in the suburbs?” If the argument in the text is correct, this is a question which Mrs. Robinson's apparatus cannot answer.
3  The legal concept of “employer and employee” and the economic concept of a firm are not identical, in that the firm may imply control over another person's property as well as over their labour. But the identity of these two concepts is sufficiently close for an examination of the legal concept to be of value in appraising the worth of the economic concept.
1  Batt, The Law of Master and Servant, p. 6.
2  Op. cit., p. 7.
